packages/web/pages/contact-us.js

```diff
@@ -0,0 +1,206 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { Title } from '../components/Common';
+import { Form, InputField, SelectField, TextField } from '../components/Form';
+import { Button } from '../components/Modal/BeAReviewerModal/styles';
+import { useModal } from '../hooks';
+
+const contactOptions = [
+	{ id: 1, label: 'Sugestão de melhoria', value: 'improvement-suggestion' },
+	{ id: 2, label: 'Reportar erro', value: 'report-error' },
+	{ id: 3, label: 'Reportar abuso', value: 'report-abuse' },
+	{ id: 4, label: 'Crítica', value: 'criticism' },
+];
+
+const Actions = ({ form }) => {
+	const handleReset = () => {
+		const { reset } = form;
+		reset();
+	};
+
+	return (
+		<StyledActions>
+			<Button type="submit" variant="contained">
+				Enviar mensagem
+			</Button>
+
+			<CancelButton type="button" variant="outlined" onClick={handleReset}>
+				Limpar campos
+			</CancelButton>
+		</StyledActions>
+	);
+};
+
+Actions.propTypes = {
+	form: PropTypes.shape({
+		reset: PropTypes.func,
+	}),
+};
+
+Actions.defaultProps = {
+	form: {},
+};
+
+const inputWrapperCss = css`
+	margin-top: 1.6rem;
+`;
+
+const ContactUs = () => {
+	const { openModal } = useModal();
+
+	const handleSubmit = () => {
+		openModal('contactUsSuccess');
+	};
+
+	return (
+		<Wrapper>
+			<Container>
+				<Form onSubmit={handleSubmit} defaultValues={{ user: { contactReason: '' } }}>
+					<StyledTitle align="left" noPadding noMargin>
+						Fale conosco
+					</StyledTitle>
+					<InputField
+						name="user.name"
+						type="text"
+						label="Qual o seu nome?"
+						validation={{ required: true }}
+						variant="gray"
+					/>
+
+					<InputField
+						name="user.email"
+						type="email"
+						label="Seu melhor e-mail"
+						validation={{ required: true }}
+						wrapperCss={inputWrapperCss}
+						variant="gray"
+					/>
+
+					<InputField
+						name="user.phone"
```

Comment:
I think we might consider using the `MaskedInputField` here to turn this into a masked input. There is a good example here: https://github.com/ufersa/plataforma-sabia/blob/2a232df29fa3fd4b59162edaf40896a2cba5b56a/packages/web/components/TechnologyForm/Responsible/index.js#L123. Take a look to see if it's not too much trouble...if not, switch it for the MaskedInputField 👍

Reply:
Easy peasy, thanks for suggesting that 😛 

![Oct-29-2020 14-14-28](https://user-images.githubusercontent.com/13091635/97608465-28f6ae80-19f1-11eb-9cad-87cea7956f94.gif)